package.json

```diff
@@ -132,7 +133,7 @@
         "karma-firefox-launcher": "^1.2.0",
         "karma-mocha": "^1.3.0",
         "karma-spec-reporter": "0.0.32",
-        "lerna": "^3.18.3",
+        "lerna": "^3.20.2",
```

Comment:
What exactly was the reason to bump two minor versions of lerna? 🤔

Reply:
The lerna project is quite good about adhering to semver, so this should only pick-up bug fixes and new featuresl

Reply:
Yep, I agree and know.. just thought it is out of the scope of this PR idk (4.1 contribution rules) :-)

Reply:
For additional background: the existence of several moderate/high vulnerability audit warnings originating in Lerna was flagged by a longtime contributor in this [2744 comment, yesterday][1]. 

[1]: https://github.com/ethereum/web3.js/issues/2744#issuecomment-613590354